spec/models/paypal_record_spec.rb

```diff
@@ -3,6 +3,29 @@
 require 'rails_helper'
 
 RSpec.describe PaypalRecord do
+  describe 'status mappings' do
+    it 'contains all wca_statuses' do
+      expect(PaypalRecord::WCA_TO_PAYPAL_STATUS_MAP.keys.sort.map(&:to_s)).to eq(PaymentIntent.wca_statuses.values.sort)
+    end
+
+    it 'contains all paypal_statuses' do
+      expect(PaypalRecord.paypal_statuses.keys.sort).to eq(PaypalRecord::WCA_TO_PAYPAL_STATUS_MAP.values.flatten.sort)
+    end
+  end
+
+  describe 'validates stripe_status' do
```

Comment:
Should be named `validates paypal_status`

Reply:
Aaaargh copy-pasta :spaghetti: